scripts/managed-services/get-kafka-instance-state/get-kafka-instance-state.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
```

Comment:
enable these bash options (bash best practice)

```
set -e
set -o nounset
set -o pipefail

```

Reply:
I tested this by scaling down my statefulSet and trying to run the command - it stopped it from running which meant that the logs on the oc exec commands timing out were lost. Removing `set -e` and adding some extra checks has solved that problem. What do you think of the updated version? Should I add `set -e` back, even if it means some logs/info in this situation may be lost?

Reply:
On reflection, this script is okay without the `-e` flag.